crutest/src/cli.rs

```diff
@@ -444,13 +445,18 @@ async fn handle_dsc(
                 println!("Got res: {:?}", res);
             }
         }
-    } else if dsc_cmd == DscCommand::Connect {
-        let url = "http://127.0.0.1:9998".to_string();
-        println!("Connect to {:?}", url);
-        let rs = Client::new(&url);
-        *dsc_client = Some(rs);
     } else {
-        println!("dsc: Need to be connected first");
+        match dsc_cmd {
```

Comment:
could you write this as
```rust
} else if let DscCommand::Connect { server } = dsc_cmd {
```
instead of the nested `match`?

Reply:
Yes, that's much better.  Done in 11fb248